lib/mayaUsd/python/wrapStageCache.cpp

```diff
@@ -33,7 +33,7 @@ void wrapStageCache()
     class_<UsdMayaStageCache>("StageCache")
 
         .def("Get", &UsdMayaStageCache::Get,
-             (args("forcePopulate") = true),
+             args("loadAll"),
```

Comment:
By removing the default = true, doesn't that mean that anyone who uses the python Get() method now has to provide an argument?

Reply:
Yes exactly. No callers in C++ ever set the parameter and some were using it wrong, so my assumption is python callers will be having the same type of error. Where should I be searching for existing python scripts to update? I don't want to break existing user scripts if I can avoid it.

Reply:
I would search this maya-usd repo.

Reply:
Good catch, @seando-adsk. This is indeed technically an API breaking change for Python clients. Thanks @williamkrick for updating the usage in this repo. We have internal `StageCache` usages that we'll need to update as well though. I'm not sure whether we have a good mechanism for things like this to inform people when updates to client code might be required.